### PR TITLE
860218 - Added UG Documentation for New .NET MAUI SfSwitch Control

### PR DESCRIPTION
**Pull request by PriyadharshniM**
Hi Team,

In this PR I have added UG document content for SfSwitch control. Please review and merge the same.

Regards,
Priyadharshni M

### CODE REVIEW

**Review by M-A32 — CHANGES_REQUESTED** [review]
I have completed the review.

File: MAUI/Switch/states.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![switch control displaying disabled off state](images/States/Disabled-Off.png)
+
+## Disabled Indeterminate
+
+The disabled indeterminate state can be enabled when you need to display the work progress. The below code example demonstrates loading the switch in disabled indeterminate state by setting [`IsOn`] property value as null and `IsEnabled` property as false.
```
[the remark this answers is not in the document]
Comment (PriyadharshniM) [reply MAUI/Switch/states.md]: I've updated all the contents.

File: MAUI/Switch/Events.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+{% highlight xaml %}
+
+<syncfusion:SfSwitch StateChanged="SfSwitch_StateChanged" StateChanging="SfSwitch_StateChanging"/>
+	
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfSwitch sfSwitch = new SfSwitch();
+sfSwitch.StateChanged += SfSwitch_StateChanged;
```
Comment (Choza-rajan) [inline MAUI/Switch/Events.md:78]: Remove the StateChanged in here

Comment (PriyadharshniM) [reply MAUI/Switch/Events.md]: Removed StateChanged

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**